frontend/src/pages/[user]/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { SWRConfig } from 'swr'
+import useSWRInfinite, { unstable_serialize } from 'swr/infinite'
+
+import Page from '@components/Page'
+import CardsGrid, {
+  getKey,
+  gridFetcher,
+  Project,
+  useUpdateBeforeReachingLimit,
+} from '@components/ProjectCardGrid'
+import { userExists } from '@utils/giteaApi'
+import { GetServerSideProps } from 'next'
+
+export const getServerSideProps: GetServerSideProps = async ({ params }) => {
+  const username = params.user as string
+  const exists = await userExists(username)
+  if (!exists) {
+    return {
+      notFound: true,
+    }
+  }
+
+  const q = {
+    query: '*',
+    filter: `ownerName = ${username}`,
+  }
+
+  const hits = await gridFetcher(q)
+
+  return {
+    props: {
+      swrFallback: {
+        [unstable_serialize(getKey(q))]: hits,
```

Comment:
This is still a little confusing. It's called `q` because in the URL it's `?q=` but you are calling that string `query` and it's part of a `q` object.

Reply:
I changed it to `searchParams`.